ext/DynaLoader/DynaLoader_pm.PL

```diff
@@ -550,6 +558,8 @@ sub dl_find_symbol_anywhere
     return undef;
 }
 
+1;
```

Comment:
I've spent a bunch of time questioning my understanding of how to make a module true. 

I think this line (much higher up) is making the module true and I get it but it still surprises me that it works. 

```
1; # End of main code
```

Reply:
This of course blames to Larry showing off in a0d0e21ea6e

Reply:
Specifically it's the file that needs to return true when required. The sub definitions are evaluated at parse time so don't affect the "return value".